Competition2017/src/telemetry/InfluxDBConnection.java

```diff
@@ -0,0 +1,48 @@
+package telemetry;
+
+import java.util.concurrent.TimeUnit;
+
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Point;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.StringProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class InfluxDBConnection {
+
+    protected final BooleanProperty isEnabledProp;
+    protected final StringProperty influxAddressProp;
+    
+    protected final InfluxDB influxDB;
+    protected final String dbName = "XbotRobotData";
+    
+    @Inject
+    public InfluxDBConnection(XPropertyManager propMan) {
+        isEnabledProp = propMan.createPersistentProperty("InfluxDB logging enabled", false);
+        influxAddressProp = propMan.createPersistentProperty("Influx address", "http://10.4.88.100:8086");
```

Comment:
It was indeed a very cool project you did and I agree we could roll something ourselves here if we wanted to. That said, this relatively off-the-shelf solution seems to be promising and something being fun/cool is a reasonable reason to pursue something on xbot from (my perspective). 

I think we disagree about the merits of the current solution compared to what would be involved with rolling our own at this point in time (and that's okay). If you're interested in working on a solution here either now or in the summer I would certainly support it (and I'm sorry I don't remember the summer proposal you were referring to before that was shut down).